chain/tlbc/tlbc-spec.json

```diff
@@ -9,16 +9,51 @@
+                                "0x11B1785D9Ac81480c03210e89F1508c8c115888E",
+                                "0x1A35cAC5aD61ED5AD42aE322cD71b915bddB6619",
+                                "0x22118560A79eb6b8f2AE267D64480c06DD1Cb243",
+                                "0x236dAA98f115caa9991A3894ae387CDc13eaaD1B",
+                                "0x2BAb548B6F7C07d5Bc72c342FB587Ca669628a20",
+                                "0x3343AD46C600a1c862A60FD5ce73396DdB43a0b6",
+                                "0x33b9Bab2B8c094a42B4E50107ea477a2a2A66C93",
+                                "0x3F5948DE91A72e6885F18EEC428F465d65e5A686",
+                                "0x3a16c4Ae8c82ebFB0AB475bB83C4BeD1c421aAa6",
+                                "0x421c65B17a331deFee0aa401fe2541F8D1cAf46D",
+                                "0x4386Eb681DBf8b97A70577dDF46415EC4992c257",
+                                "0x47C2F6f13C97C4a4D36B4E695F26d648Ee053ec4",
+                                "0x640D34E204Be6cbf8846D68C797971AAa4F91a98",
+                                "0x663d3947f03eF5B387992b880aC85940057c13e3",
+                                "0x68E1B4E5d789B1BE5c2de5990922C960B4Bbdb9a",
+                                "0x6a0CD7ec77a3014C35dbE2F8D18A512E962BDc47",
+                                "0x743E2e09c20BcC93D7C2dA47DD0b5d1DA3A35fCf",
+                                "0x74dc2D7B785C89057b862d70b308B91Bd2B1D0A4",
+                                "0x83fb539DF9CBb74Ac642CA492308028819570eA9",
+                                "0x8C920861A976E125fA3664e35E91D4c73a90C8C9",
+                                "0x8Fd22E50Bd4dFCC28DDC2412c8b3fD8EF7835BF3",
+                                "0x93903dCbB0f0bE5Ac528238E9a10F642e401D569",
+                                "0x94BeE20aC9b9645d533a0bdf14fEB7bc8CDfCE30",
+                                "0x9CBFC85263604529Dac5F6512985067331933785",
+                                "0x9F20e2468F37A319992403B953cEB218CA9fE79a",
+                                "0x9e557934B1f2488D37c1F739aC34eE1907C562C0",
+                                "0xA63631814773DC207ec396cDba84D0cC40DB3788",
+                                "0xA82289545e7a75aF8618c1Bd6816C26e3a8AE09c",
+                                "0xBA594f053bE04Ee8942d0d3fA05766F49C5516F8",
+                                "0xD112F8122dCcDd64e18dbA537aC269F0F98B03B3",
+                                "0xD31e0eEC136353bc77b49d1FEE27292A8b5bdc93",
+                                "0xE21C2BB2d45434892C23059a32EDc6eC8108D0c4",
+                                "0xEF18997A4189a72952051B5bBf1DbE471403425c",
+                                "0xEF1899d129f2dDF103cd94475DFd87A1D433f284",
+                                "0xEdbFa87722F7EefF9D711Ca5b084C6c00CA0765f",
+                                "0xFda153D9ad9c51a5637F96C82427510bd8202847",
+                                "0xb4CA1c77518665e56B2c0c1ae7F1c56b9fC7Db0a",
```

Comment:
Sorting the addresses case-insensitively would make more sense IMO.

Reply:
Done